cli/utils.go

```diff
@@ -108,18 +105,19 @@ func (cmd BuildFlags) saveImage(imageName image.Name, imageStore storage.ImageSt
 }
 
 // cleanManifest removes specified image manifest from local filesystem.
-func (cmd BuildFlags) cleanManifest(imageName image.Name, imageStore storage.ImageStore) error {
+func (cmd BuildFlags) cleanManifest(imageName image.Name) error {
 	repo, tag := imageName.GetRepository(), imageName.GetTag()
-	if err := imageStore.Manifests.DeleteStoreFile(repo, tag); err != nil && !os.IsNotExist(err) {
```

Comment:
This looks alright... Is goreport complaining about this line?

Reply:
The line became > 100 characters when i added the `cmd.` in front of the imageStore, i'd rather split the if into a new line now if that's not an issue with you.